### ISSUE TITLE
wx._core.wxAssertionError upon executing

### ISSUE THREAD

**Issue by Rapptz** [hasegaw/IkaLog#108]
Commit: 33bf400

Traceback:

```
Traceback (most recent call last):
  File "IkaUI.py", line 44, in <module>
  File "Z:\hasegaw\work\IkaLogPY2EXE\ikalog\ui\capture.py", line 257, in on_option_tab_create
wx._core.wxAssertionError: C++ assertion "N == (-1) || IsValid(N)" failed at ..\..\src\msw\listbox.cpp(286) in wxListBox::DoSetSelection(): invalid index in wxListBox::SetSelection
running destructor
```

From my understanding this occurs because the number of cameras returned in `VideoCapture.enumerate_devices` returns an empty list in here: https://github.com/hasegaw/IkaLog/blob/master/ikalog/ui/capture.py#L58

Reasons for which I'm not sure of.

**Comment by hasegaw** [comment]
Thanks for reporting. Seems no any input devices are detected. Just want to know if you have any cameras or input devices on the system? Try any webcams attached if you have.

**Comment by Rapptz** [comment]
I have a capture card (an Elgato HD60). However I don't have a webcam to test that.

**Comment by hasegaw** [comment]
Unforunately Elgato HD60 seems not to be detected as DirectShow input device. Installing a virutal camera software (e.g. XSplit) can be a workaround.

**Comment by Rapptz** [comment]
Ah. That's a shame. Thanks.

**Comment by hasegaw** [comment]
You still have a chance to grab the image of bundled software using IkaLog's ScreenCapture once it started. XSplit can be used for bridging software. See en_WinIkaLog page in the Wiki for details.

**Comment by hasegaw** [comment]
Fixed in 789fb2af8a05268040bee5965174fb2b54947e11
